umh-core/pkg/service/protocolconverter/runtime_config/runtime_config.go

```diff
@@ -293,6 +295,9 @@ func renderConfig(
 		return protocolconverterserviceconfig.ProtocolConverterServiceConfigRuntime{}, err
 	}
 
+	// the downsampler will be appended only for timeseries data
```

Comment:
I think the project here should be now, if data type equals time series, then append downsampler and not put it in the append downsampler function, but instead put it here and then append downsampler can just always append downsampler. This makes it a little bit more intuitive, but should be done.